.github/workflows/tests.yaml

```diff
@@ -15,6 +15,8 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu, windows, macos]
+    env:
+      PYTHONWARNINGS: ignore
```

Comment:
A few deprecation warnings, some for Py2 & some for Py3. Out of scope to fix those in this PR, so will just add suppression here.